doc_user/UserManual.adoc

```diff
@@ -764,6 +765,33 @@ other lots can satisfy the request. Ask the Administrator or check the _Administ
 
 Before pressing the *[.underline]##O##K* button, insert as many Medicals as needed; it will show a corresponding number of movements in the *Outcomes* tab of the *Movement panel* in the *_Ward Pharmacy_* window. These movements are also visible in the *Incomings* tab of the *Movement panel* in the *_Ward Pharmacy_* window of the receiving ward, in case of discharging to another ward.
 
+===== 4.3.2.5 Delete the Last Stock Movement in a Ward ([.underline]##D##elete)
+In pharmaceutical stock ward, we have possibility to discharge medical from *patient*, *internal use* or *another ward*. if you make a mistake during the discharge you have the possibility to delete it in order to correct the stock of the ward and resume the action if possible.
+
+Nevertheless, the admin can enable/disable the *[.underline]##D##elete* button that allows the enabled user do delete only the last movement, and helps quickly undoing mistakes.
+
+To delete the last movement, select first the row that you want to delete and press the Delete button in the Stock Movement Browser window. Only one row is allowed to delete at a time.
+
+If it is not the last movement the following Error message is shown:
+
+image:image250.png[Not last movement error]
+
+But if the row selected is the last movement this *_Question_* message is presented:
+
+image:image248.png[Question]
+
+Click *[.underline]##Y##es* to confirm the deletion of the last movement, otherwise click *[.underline]##N##o* to cancel the operation. After clicking the *[.underline]##Y##es* button, the system checks if the product involved in the selected movement has not been used itself in the *_Pharmaceuticals Stock [.underline]##W##ard_* if the last movement is discharge from ward to ward.
```

Comment:
```suggestion
Click *[.underline]##Y##es* to confirm the deletion of the last movement, otherwise click *[.underline]##N##o* to cancel the operation. After clicking the *[.underline]##Y##es* button, the system checks if to see if the product involved in the selected movement and has not been used in the *_Pharmaceuticals Stock [.underline]##W##ard_*.   If not then the last movement is discharged from ward to ward.
```